src/mlpack/tests/perceptron_test.cpp

```diff
@@ -23,6 +24,61 @@ using namespace mlpack::distribution;
 BOOST_AUTO_TEST_SUITE(PerceptronTest);
 
 /**
+ * This test tests whether the SimpleWeightUpdate updates weigths and biases correctly.
+ */
+BOOST_AUTO_TEST_CASE(SimpleWeightUpdateUpdateWeights)
+{
+  SimpleWeightUpdate wip;
+
+  vec trainingPoint("1 2 3 4 5");
+  mat weights("0 1 6;"
+              "2 3 6;"
+              "4 5 6;"
+              "6 7 6;"
+              "8 9 6");
+  vec biases("2 5 7");
+  size_t incorrectClass = 0;
+  size_t correcClass = 2;
```

Comment:
Do you mind adding a comment somewhere in here about what `UpdateWeights()` is supposed to do?  (In this case, add the point to the correct class weights and subtract the point from the incorrect class weights.)